mllib/src/test/scala/org/apache/spark/mllib/evaluation/RegressionMetricsSuite.scala

```diff
@@ -22,91 +22,115 @@ import org.apache.spark.mllib.util.MLlibTestSparkContext
 import org.apache.spark.mllib.util.TestingUtils._
 
 class RegressionMetricsSuite extends SparkFunSuite with MLlibTestSparkContext {
+  val obs = List[Double](77, 85, 62, 55, 63, 88, 57, 81, 51)
+  val eps = 1E-5
 
   test("regression metrics for unbiased (includes intercept term) predictor") {
```

Comment:
The old test passed after the changes I made except for the r^2 (as expected). The reason I changed the test completely was that I couldn't get the value of r^2 from R without fitting the model. I didn't want to copy/past my formula in R and get the results from there. I really wanted to get the rest of parameters also from the fit in R, but that can only be dome by using anova table which is cumbersome.